lib/opencensus/stats/aggregation_data/distribution.rb

```diff
@@ -50,14 +53,16 @@ def initialize buckets
           @mean = 0
           @sum_of_squared_deviation = 0
           @bucket_counts = Array.new(buckets.length + 1, 0)
+          @exemplars = []
           @start_time = Time.now.utc
         end
 
         # @private
         # Add value to distribution
         # @param [Integer,Float] value
         # @param [Time] time Time of data point was recorded
-        def add value, time
+        # @param [Hash<String,String>] attachments
```

Comment:
If `nil` is a valid type, include that in the type spec. Also, include a description of the attachments parameter.